ribbon/src/main/java/feign/ribbon/LBClientFactory.java

```diff
@@ -0,0 +1,25 @@
+package feign.ribbon;
+
+import com.netflix.client.ClientFactory;
+import com.netflix.client.config.IClientConfig;
+import com.netflix.loadbalancer.ILoadBalancer;
+
+/**
+ * Factory to be used for the creation of {@link LBClient}'s for usage in {@link RibbonClient}
```

Comment:
I appreciate the polish, but I'm not sure this clarifies much. Let's remove javadoc that don't pull their weight.